lib/takeaway.rb

```diff
@@ -0,0 +1,43 @@
+require './lib/menu'
+require './lib/message'
+require './lib/order'
+
+class Takeaway
+
+  attr_reader :order
+
+  def initialize(filename='./lib/menu.csv')
+    @menu_file = filename
+    @menu = Menu.new(filename)
+  end
+
+  def print_menu
+    @menu.read_menu
+  end
+
+  def please_may_i_order
+    @order = Order.new(@menu_file)
+    "Sure, go right ahead!"
```

Comment:
Could the logic for talking to the customer be separate from the logic for making the order?